.github/workflows/new-cfe-cherry-pick.yml

```diff
@@ -0,0 +1,205 @@
+name: New CFE workflow - Cherry pick
+on:
+    pull_request:
+        types: [closed]
+    workflow_dispatch:
+        inputs:
+            skipSlackPing:
+                description: 'Skip Slack Ping: If true, the Slack ping will be skipped (useful for testing)'
+                type: boolean
+                required: false
+                default: false
+
+env:
+    GIT_COMMITTER_NAME: 'WooCommerce Bot'
+    GIT_COMMITTER_EMAIL: 'no-reply@woocommerce.com'
+    GIT_AUTHOR_NAME: 'WooCommerce Bot'
+    GIT_AUTHOR_EMAIL: 'no-reply@woocommerce.com'
+
+permissions: {}
+
+jobs:
+    verify:
+        name: Verify
+        runs-on: ubuntu-20.04
+        outputs:
+            run: ${{ steps.check.outputs.run }}
+            base_ref: ${{ steps.fetch_pr_details.outputs.base_ref }}
+        steps:
+            - name: Fetch Pull Request Details
+              if: github.event.pull_request
+              id: fetch_pr_details
+              uses: actions/github-script@v6
+              with:
+                script: |
+                  const pullRequestUrl = context.payload.pull_request.url;
+                  const prDetails = await github.request(pullRequestUrl);
+                  core.setOutput('base_ref', prDetails.data.base.ref);
+              env:
+                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+            - name: check
+              id: check
+              uses: actions/github-script@v6
+              with:
+                  script: |
+                      const baseRef = process.env.BASE_REF;
+                      console.log("baseRef:", baseRef);
+
+                      let run = false;
+
+                      const isBot = context.payload.pull_request && ( context.payload.pull_request.user.login == 'github-actions[bot]' || context.payload.pull_request.user.type == 'Bot' );
+                      console.log("isBot:", isBot);
+                      console.log("baseRef.startsWith('release/'):", baseRef.startsWith('release/'));
+
+                      if ( !isBot && baseRef.startsWith( 'release/' ) ) {
+                        core.setOutput( 'run', 'true' );
+                      } else {
+                        core.setOutput( 'run', 'false' );
+                      }
+              env:
+                BASE_REF: ${{ steps.fetch_pr_details.outputs.base_ref }}
+
+    prep:
+        name: Prep inputs
+        runs-on: ubuntu-20.04
+        if: needs.verify.outputs.run == 'true'
+        needs: verify
+        outputs:
+            release: ${{ steps.prep-inputs.outputs.release }}
+            pr: ${{ steps.prep-inputs.outputs.pr }}
+            version: ${{ steps.prep-inputs.outputs.version }}
+        steps:
+            - name: Prep inputs
+              id: prep-inputs
+              uses: actions/github-script@v6
```

Comment:
Is it worth bumping this to `@v7`? Might head off warnings like this one:

> The following actions use a deprecated Node.js version and will be forced to run on node20:  
>  
> actions/github-script@v6.  
>  
> For more info: https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/

Reply:
Good catch, added it here: https://github.com/woocommerce/woocommerce/pull/51207/commits/739c2b7b6874ff032a71c734a8e56880bce4af72